pkgs/development/libraries/openssl/default.nix

```diff
@@ -205,4 +205,16 @@ in {
     ];
     withDocs = true;
   };
+
+  openssl_3_0 = common {
+    version = "3.0.0";
+    sha256 = "sha256-We7fy0bCUhTJvTftYHgpe03wHQEiZ/6enu4x9hvHBTY=";
+    patches = [
+      ./3.0/nix-ssl-cert-file.patch
+      (if stdenv.hostPlatform.isDarwin
+       then ./1.1/use-etc-ssl-certs-darwin.patch
+       else ./1.1/use-etc-ssl-certs.patch)
```

Comment:
I'd rather we duplicate the patches into a `3.0` directory, since this looks very error-prone.

Reply:
Or we move them up one directory.